.github/workflows/pull-requests.yml

```diff
@@ -39,7 +39,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-         version: [ 20.10.5-buster-slim, 21.10.1-buster-slim, ci ]
+         version: [ previous-lts, lts, latest ]
```

Comment:
why did this change? my PR also had some trouble running tests on 20.10.5-buster-slim, may be it's related to your fix...

Reply:
I don't know why `20.10.5-buster-slim` stopped working all of the sudden. However, on the other PR I moved to `previous-lts`, `lts` and `latest` because that's what I should have done a while ago. I didn't know we pulled the trigger on the new naming.

Reply:
Done